### DISCUSSION TITLE
No root mode webview

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#970] [Ideas]
Hey i found some ideas we still can change webview from dev option no root with this following method:

https://www.stoutner.com/geckoview/

From Link Above:
Webview that have same Codebase from google webview are able to used WITHOUT ROOT
......

![IMG_20210329_140712](https://user-images.githubusercontent.com/50813528/112799450-912ce780-9098-11eb-8051-43fe572256b1.jpg)

@csagan5

**Comment by ghost** [comment]
> Hey i found some ideas we still can change webview from dev option no root with this following method:

Unfortunately bromite or any other webview app aren't showing in developer settings to change ;\

**Reply by ghost** [reply]
Did you read the link that i gave you?

If your Webview have SAME CODEBASE FROM GOOGLE WEBVIEW then you can change it without root 

![IMG_20210329_140712](https://user-images.githubusercontent.com/50813528/112799155-2b406000-9098-11eb-8808-e8ee21762ad8.jpg)



BRUH

**Reply by ghost** [reply]
For example??

**Reply by ghost** [reply]
> For example??

Check link that i post on 1st thread bro

**Reply by csagan5** [reply]
It is not possible to install a custom webview without root

**Reply by ghost** [reply]
> It is not possible to install a custom webview without root

it's possible sir please read my link webpage on 1st post

**Reply by uazo** [reply]
no, trust me, it's not possible, i saw in the android source how it works.

what the author writes in that link is about the choice of the webview in HIS app, because being the webview a library it can decide which webview to use in HIS process, but IT IS NOT POSSIBLE (unfortunately) to change the system-wide one without modifying files untouchables of android, which are editable only with root.

**Reply by csagan5** [reply]
> IT IS NOT POSSIBLE (unfortunately)

It is a security measure; everything is explained here: https://github.com/bromite/bromite/wiki/Installing-SystemWebView#installation-without-root

**Reply by ghost** [reply]
@uazo 

i mean if your webview have same code base from google you still can change it from dev setting even app installed as normal (user) 

that's all the web tell us

**Reply by uazo** [reply]
no, android checks the digital signature (if configured in config_webview_packages.xml) and if it does not match it does not allow its use.
please also read https://github.com/bromite/bromite/issues/520
